### ISSUE TITLE
Nav Unification: Media Library won't scroll search results, possibly due to number of results

### ISSUE THREAD

**Issue by macmanx2** [Automattic/wp-calypso#51695]
### Steps to reproduce the behavior

1. Go to https://wordpress.com/media
2. Search for a term that will return more images that one page of results.
3. Scroll down.

#### What I expected to happen

Images will continue to scroll through results.

#### What actually happened

On Eszv-p2 when searching for "Michigan," of which there are 602 results, scrolling simply stops after 20.

On my own test site where my highest matching result is only 278, scrolling went through all images.

### Context

Reported via https://wordpress.com/forums/topic/image-library-search/

#### Browser / OS version

Does not appear to be browser or OS-specific.

#### Is there any console output or error text?

Nothing that appears to be related.

#### Level of impact (Does it block purchases? Does it affect more than just one site?)

Appears to only impact sites with excessive amount of possible results.

#### Reproducibility (Consistent, Intermittent) Leave empty for consistent.

Consistently reproducible on reporting site.

#### Screenshot / Video: If applicable, add screenshots to help explain your problem.

<img width="1361" alt="Screen Shot 2021-04-05 at 2 13 29 PM" src="https://user-images.githubusercontent.com/4856235/113628563-e7aaa080-9619-11eb-8dea-996891c163ef.png">

**Comment by obenland** [comment]
@tyxla Is it possible this could have been caused by https://github.com/Automattic/wp-calypso/pull/51681?

**Comment by tyxla** [comment]
> @tyxla Is it possible this could have been caused by #51681?

I don't think so. That affected only the date that is displayed for each group, but nothing more. 

I'm actually unable to reproduce - tested with multiple sites - WP.com simple, Jetpack, Atomic, and couldn't repro with any of them. Is it possible there is an error visible in the browser console or in an API request response?
